addons/sale_project_mrp/models/mrp_bom.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+
+class MrpBomLine(models.Model):
+    _inherit = 'mrp.bom.line'
+
+    is_service = fields.Boolean('Is a service', compute='_compute_is_service',
```

Comment:
if not for its project related help message, shouldn't this field be better to put it into `sale_mrp`? In fact, it could go directly into `mrp` module instead.

Reply:
The field is only useful in the bridge module. Therefore, i'd rather have it in here so that the field is created in the database only if necessary.